rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java

```diff
@@ -143,6 +92,8 @@
     private String login;
     private String password;
 
+    private boolean initialized = false;
+
     private SchedulerEventReceiver schedulerEventReceiver;
     
     private static final Logger logger = ProActiveLogger.getLogger(SchedulerClient.class);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename the "logger" logger to comply with the format "LOG(?:GER)?". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1312)